cmd/snap/cmd_find.go

```diff
@@ -100,6 +102,10 @@ func (x *cmdFind) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
+	if x.Positional.Query == "" {
+		return errors.New(i18n.G("you need to specify a query. Try \"snap find hello-world\"."))
```

Comment:
Yeah, we use it elsewhere and often in places where it would be very awkward to have something else (most likely: an absolute path to the binary instead of its bare name). And we haven't actually changed the name of a binary before; we added a new one, and removed the old one. Here's hoping we do it that way again if we do do it again.